src/mlpack/bindings/julia/mlpack/cli.jl.in

```diff
@@ -1,29 +1,29 @@
 module cli
```

Comment:
I haven't found all the filenames that need to change, but I've found a few and labeled them.  Do you think you could take a look through and handle any `cli` -> `io` filename changes that are needed?